Learn-CSS-Blog/index7.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Home | Learn Basic CSS by building a Blog</title>
+</head>
+<body>
+    <div id="main"></div>
+</body>
+</html>
+
+<!-- 
+    Nest a `header` element inside of the first `<div>`.
```

Comment:
No need for an example here:

    Nest a `<header>` element inside of the first `<div>`.